components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineSession.kt

```diff
@@ -312,7 +312,8 @@ class SystemEngineSession(private val defaultSettings: Settings? = null) : Engin
 
         return webView?.let {
             it.buildDrawingCache()
-            val outBitmap = Bitmap.createBitmap(webView.drawingCache)
+            val drawingCache = webView.drawingCache ?: return null
```

Comment:
We'll have to wait until someone else can come by and merge this, but till then, one-liners for null checks are always nice.
```kotlin
val outBitmap = webView.drawingCache?.apply { Bitmap.createBitmap(this) }
```

No need to fix if you have other things to do.

Reply:
Good call, updating.